### DISCUSSION TITLE
Pro version - able to save all tracks separately

### DISCUSSION THREAD

**Discussion by anttiryt** [jamulussoftware/jamulus#1757] [Ideas]
Us and I think many other choirs and companies could use and pay for a pro version. Some features I can think of

* save all tracks
* play a mp3 file, seek bar, start/stop, notes on positions
* other mixer tools
* all saved on server for future use

**Comment by ann0see** [comment]
Hi! Glad to hear you like Jamulus! 
Providing a Pro version is probably not what we all want ;-). Jamulus is Free and Open Source Software which objects adding restrictions like paid only features. 

However, everyone is free to add features as he/she wants. 

So you could either add a feature yourself and request to include it into the official version or wait for a volunteer to do it for you. 

I even think some of this options already exist:

> save all tracks

Quite easy: enable recording on the server. This gives you .wav files for each user. Also you can open the files in Audacity or Reaper. See https://jamulus.io/wiki/Server-Win-Mac

> play a mp3 file, seek bar, start/stop, notes on positions

Although that’s a bit more tricky, it’s not impossible. I usually use JACK on Linux to get audio from my DAW (Ardour) and play it to Jamulus. Adding mp3 support directly is a big task and would make Jamulus a lot more complicated (not only code wise). We already suffer from being complicated in the setup process ;-(.

> other mixer tools

What do you think of specifically?
